# srv/GetBestNodeForExploration.srv
nav_msgs/OccupancyGrid maphttps://github.com/sathwikkannam/r-rune-os2/edit/main/project/srv/Explore.srv
---
int32[] best_node
